vmdb/spec/migrations/20150316175916_update_miq_database_default_update_repo_name_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "spec_helper"
+require Rails.root.join("db/migrate/20150316175916_update_miq_database_default_update_repo_name")
+
+describe UpdateMiqDatabaseDefaultUpdateRepoName do
+  let(:db_stub) { migration_stub(:MiqDatabase) }
+
+  migration_context :up do
+    [
+      "Satellite 5 without scl repo", "rhel-x86_64-server-6-cf-me-3",                                "rhel-x86_64-server-6-cf-me-3.2 rhel-x86_64-server-6-rhscl-1",
+      "Satellite 5 with scl repo",    "rhel-x86_64-server-6-cf-me-3.1 rhel-x86_64-server-6-rhscl-1", "rhel-x86_64-server-6-cf-me-3.2 rhel-x86_64-server-6-rhscl-1",
+      "Satellite 6",                  "cf-me-5.3-for-rhel-6-rpms rhel-server-rhscl-6-rpms",          "cf-me-5.4-for-rhel-6-rpms rhel-server-rhscl-6-rpms",
+    ].each_slice(3) do |name, old, new|
+      it name do
+        db = db_stub.create!(:update_repo_name => old)
+
+        migrate
+
+        expect(db.reload.update_repo_name).to eq(new)
+      end
+    end
+  end
+
+  migration_context :down do
+    [
+      "Satellite 5", "rhel-x86_64-server-6-cf-me-3.2 rhel-x86_64-server-6-rhscl-1", "rhel-x86_64-server-6-cf-me-3.1 rhel-x86_64-server-6-rhscl-1",
+      "Satellite 6", "cf-me-5.4-for-rhel-6-rpms rhel-server-rhscl-6-rpms",          "cf-me-5.3-for-rhel-6-rpms rhel-server-rhscl-6-rpms",
+    ].each_slice(3) do |name, old, new|
```

Comment:
the part that is confusing is that the "3.2" repo is considered OLD while "3.1" is NEW.  In this context, old/new refers to the "existing value" vs. "desired value"